codegen/src/main/java/software/amazon/awssdk/codegen/poet/auth/scheme/AuthSchemeParamsSpec.java

```diff
@@ -107,6 +112,18 @@ private void addAccessorMethods(TypeSpec.Builder b) {
                                               + "with $S auth scheme. By default, the region will be empty.", "aws.auth#sigv4")
                                   .build());
         }
+
+        if (authSchemeSpecUtils.generateEndpointBasedParams()) {
+            parameters().forEach((name, model) -> {
+                if (authSchemeSpecUtils.includeParam(name)) {
+                    MethodSpec accessor = endpointRulesSpecUtils.parameterInterfaceAccessorMethod(name, model);
+                    if (model.getDocumentation() != null) {
+                        accessor = accessor.toBuilder().addJavadoc(model.getDocumentation()).build();
+                    }
```

Comment:
Is this because generated endpoint params doesn't current add docs, but we are trying to add them here? Feels like the answer should be same for both. Maybe we should check with team (@dagnir?) on if docs can/should be added to endpoint params.

Reply:
Yeah, I can add to the endpoints params as well, not sure I will confirm with @dagnir and/or @millems

Reply:
Weird that we don't add them, imo. Dongie can confirm when we get back why we didn't do it.